algebird-core/src/main/scala/com/twitter/algebird/BloomFilter.scala

```diff
@@ -293,10 +293,10 @@ object BF {
                       uniq(src, dst, cur, spos + 1, dpos + 1)
                     }
                   }
-                val uniqVs = new Array[Int](hashvalues.length)
+                val uniqVs: Array[Int] = new Array[Int](hashvalues.length)
                 val len: Int = uniq(hashvalues, uniqVs, -1, 0, 0)
 
-                var pos = 0
+                var pos: Int = 0
```

Comment:
this is a bummer. Can we make a change that doesn't require the annotations on private types? This cannot be seen from the outside since we are returning an IntIterator

Reply:
good question ... maybe? I'll have a look